src/components/tracked-controls.js

```diff
@@ -51,7 +51,9 @@ module.exports.Component = registerComponent('tracked-controls', {
   tick: function (time, delta) {
     var mesh = this.el.getObject3D('mesh');
     // Update mesh animations.
-    if (mesh && mesh.update) { mesh.update(delta / 1000); }
+    if (mesh && mesh.update) {
```

Comment:
Updated for consistency. It seems that the pattern for a complicated line is to break and not do it all on one line.

Reply:
Can we keep this as a one liner? In modern displays aspect ratio vertical space is at a premium.

Reply:
I think this is the last comment to address and this is ready to go.